rpc/src/v1/types/confirmations.rs

```diff
@@ -38,8 +39,24 @@ impl From<helpers::ConfirmationRequest> for ConfirmationRequest {
 	}
 }
 
+impl fmt::Display for ConfirmationRequest {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		write!(f, "id: {:?}, {}", self.id, self.payload)
+	}
+}
+
+impl fmt::Display for ConfirmationPayload {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		match self {
+			&ConfirmationPayload::Transaction(ref transaction)
+				=> write!(f, "{}", transaction),
+			&ConfirmationPayload::Sign(_) => write!(f, "TODO: data"),
```

Comment:
Yeah, not sure how this should be displayed or where/how it will be used. Suggestions?